src/components/App.tsx

```diff
@@ -39,9 +39,13 @@ class App extends React.Component<Record<string, unknown>, AppState> {
     // TODO: fetch real ETH price
     const ethPriceUsd = 2700;
     this.fetchAave(this.mainAccount, ethPriceUsd);
+
+    // deposit to Aave lending pool
+    let user = await this.web3.eth.getAccounts();
+    deposit(this.web3.eth.currentProvider, user[0]);
```

Comment:
this is now triggered automatically but should be done on demand, ie through a button click